DDDEastAnglia/Areas/Admin/Views/Sponsor/Delete.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@model DDDEastAnglia.Models.Sponsor
+
+@{
+    ViewBag.Title = string.Format("Delete {0}?", Model.Name);
+}
+
+<h2>@ViewBag.Title</h2>
+
+<p>Deleting a sponsor cannot be undone. Are you <em>really</em> sure you want to delete this sponsor?</p>
```

Comment:
I agree - there was several bits in making the CRUD for sponsors that were very similar to those for other object types. Something to be done later though, otherwise this pull request will get huge.